app/directives/reg-open.js

```diff
@@ -71,14 +71,13 @@ function setScopeProps (props){
 }
 
 function findOpenRegsQuery(){
-
   return {
             q:  {
                   '$or'                      : [ { institution: 'CBD' }, { institution: 'cbd' }],
                   schedule                   : { $exists: true },
-                  StartDate                  : { $gt: { $date: moment() } },
-                  'schedule.sideEvents.start': { $lt: { $date: moment() } },
-                  'schedule.sideEvents.end'  : { $gt: { $date: moment() } },
+                  StartDate                  : { $gt: { $date: moment.utc() } },
```

Comment:
You can use `new Date()`. Date will be serialized to utc anyway

Reply:
thanks sir

Reply:
You are correct.  Thanks.